templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -805,7 +803,7 @@ Resources:
         NodeInstanceFamily: Standard
         NodeGroupType: Managed
         NodeGroupOS: Amazon Linux 2
-        EKSClusterName: !Ref EKSClusterName
+        EKSClusterName: !Join ['-', [!FindInMap [ EKSClusterNames, Single, Name ], !Select [4, !Split ['-', !Select [2, !Split ['/', !Ref AWS::StackId]]]]]]
```

Comment:
yes that is the naming convention i was looking for. i know its a busy looking thing...wasnt sure how else to do this in a deterministic and repeatable way since mappings dont allow for the join function